src/helpers/create-pr.ts

```diff
@@ -26,17 +26,15 @@ export class CreatePR extends HelperInputs {
 export const createPr = async ({ title, body, head = context.ref.replace('refs/heads/', ''), base }: CreatePR) => {
   const pr_base = base || (await getDefaultBranch());
   await updateHeadWithBaseBranch(pr_base, head);
-  const {
-    data: { number }
-  } = await octokit.pulls.create({
+  const { data } = await octokit.pulls.create({
     title,
     head,
     base: pr_base,
     body,
     maintainer_can_modify: true,
     ...context.repo
   });
-  return number;
+  return data;
```

Comment:
Could we maybe leverage the `pull_number` input to avoid making this breaking? Like if pull_number is "false" return the whole data object? idk just spitballing

Reply:
the number will always exist as long as it succeeds, the issue is just that we arbitrarily chose to return the pull number rather than the whole pull object, which would probably make more sense

Reply:
I mean the `pull_number` input to the action, so the consumer can choose to get back the whole data object, and leave the default behavior alone. If this is too jank, we can maybe just make the breaking change but not release a v2 (I'd like to avoid doing that)

Reply:
ah no this is a good idea. i'll do that. maybe it would be more clear to use a `return_pull_number` input?

Reply:
Maybe something more generic like `return_full_payload`?

Reply:
sure